### ISSUE TITLE
Blank page on import

### ISSUE THREAD

**Issue by MeikelLP** [bnw/firefly-iii-fints-importer#92]
I'm able to collect all transactions from my bank for the given time span: 2023-04-01 until 2023-04-24

Sadly once I hit "import" I get a blank page and the app is not importing anything. I tried multiple ranges, all in April. None of them worked. I could pin it down to a single transaction having maybe a bad string in it.

Before that I successfully imported 2023-03-01 until 2023-03-31

Is there any way I can enable logging to at least see the error?

**Comment by MeikelLP** [comment]
Still fails in May. @bnw any idea?

**Comment by bnw** [comment]
Hi, sorry for the late reply. 
A crash without error message is strange. 
Are you using the docker images? The php there should be configured to at print exceptions upon crash. Does the docker container stop once you encounter the error?
If yes, do you see any logs from the command line php call? They should appear in the docker log.

**Comment by MeikelLP** [comment]
No there is no crash nor log anywhere. However with the help of a friend (I'm not a PHP dev) I was able to set it up locally and I see an error now:

```
Warning: preg_replace(): No ending delimiter '.' found in E:\projekte\Github\bnw\firefly-iii-fints-importer\app\TransactionsToFireflySender.php on line 92

Fatal error: Uncaught Exception: Error in regular expression! Match expression .* Replace expression $1 in E:\projekte\Github\bnw\firefly-iii-fints-importer\app\TransactionsToFireflySender.php:96 Stack trace: #0 E:\projekte\Github\bnw\firefly-iii-fints-importer\app\TransactionsToFireflySender.php(128): App\TransactionsToFireflySender::transform_transaction_to_firefly_request_body() #1 E:\projekte\Github\bnw\firefly-iii-fints-importer\app\RunImport.php(40): App\TransactionsToFireflySender->send_transactions() #2 E:\projekte\Github\bnw\firefly-iii-fints-importer\app\index.php(65): App\StepFunction\RunImport() #3 {main} thrown in E:\projekte\Github\bnw\firefly-iii-fints-importer\app\TransactionsToFireflySender.php on line 96
```

I may be able to fix this myself and provide a PR but please don't rely on me. I assumed it was an issue with the data from my bank. This is now confirmed due to the regex error from what I can see. I try to find out which text is causing this issue and report back.

**Comment by MeikelLP** [comment]
Well I found the error. I upgraded my importer version after my recent issue (#91) because I thought it may fix this issue. However new configurations values are "required" (I get an ugly warning in the UI if missing in json) so I set them to the defaults I thought.

However `description_regex_replace` default value is not `$1` but `$0` or `""`. This caused this issue.

Tho I'd love to have improved user experience here :)
